requirements_test_pre_commit.txt

```
@@ -1,5 +1,5 @@
 black==22.1.0
-pylint==2.12.2
+pylint==2.13
```

```suggestion
pylint~=2.13.0
```
I'm surprised 2.13 without the 0 is working ? TIL !